diff --git a/src/main.mjs b/src/main.mjs
index a8915ca74b897842f80ae005225d8236ff1b7da5..fdec895a27da26f2c16b471eb393aef8ec4c7b47 100644
--- a/src/main.mjs
+++ b/src/main.mjs
@@ -264,6 +264,7 @@ function version() {
 }
 
 function runtime() {
+  if (globalThis.Bun) return 'bun'; if (globalThis.Deno) return 'deno'; if (globalThis.HermesInternal) return 'hermes';
   if (globalThis.d8) return 'v8';
   if (globalThis.tjs) return 'txiki.js';
   if (globalThis.Graal) return 'graaljs';
@@ -278,7 +279,6 @@ function runtime() {
   if (globalThis.window && globalThis.navigator && (new Error().stack).includes('runtime@')) return 'webkit';
 
   if (globalThis.os && globalThis.std) return 'quickjs';
-  if (globalThis.Bun) return 'bun'; if (globalThis.Deno) return 'deno'; if (globalThis.HermesInternal) return 'hermes';
   if (globalThis.window && globalThis.navigator) return 'browser'; if (globalThis.process) return 'node'; else return null;
 }
 
